tests/rptest/tests/host_metrics_test.py

```diff
@@ -8,15 +8,12 @@
 # by the Apache License, Version 2.0
 
 from rptest.services.cluster import cluster
-from rptest.tests.redpanda_test import RedpandaTest
+from rptest.tests.redpanda_test import RedpandaMixedTest
 
 
-class HostMetricsTest(RedpandaTest):
+class HostMetricsTest(RedpandaMixedTest):
```

Comment:
thanks for making this swap, we should add it to the cloud test yaml though so it runs